vue/components/ui/textarea-platforme/textarea-platforme.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+    <div class="textarea-container">
+        <label-platforme
+            class="label description-label"

```

Comment:
`description-label` -> `label-description`. I know this standard is not being followed thoroughly but it should, so we end up with stuff like: `button`, `button-yellow`, `button-icon`, etc.